src/node/genesisgen.h

```diff
@@ -269,11 +286,8 @@ namespace ccf
 
     void add_key_share_info(const KeyShareInfo& key_share_info)
     {
-      auto [shares_view, values_view] =
-        tx.get_view(tables.shares, tables.values);
-      auto keyshare_id = get_next_id(values_view, ValueIds::NEXT_KEYSHARE_ID);
-
-      shares_view->put(keyshare_id, key_share_info);
+      auto shares_view = tx.get_view(tables.shares);
```

Comment:
There is no need for a separate key here. We only keep the latest version of the key share in the KV (i.e. key is `0`).